### DISCUSSION TITLE
Webhook send_message_response

### DISCUSSION THREAD

**Discussion by fahram** [kirimwa-id/forum#57] [Tanya Jawab]
Mungkin ini bug, atau mungkin kesalahan ada di saya.
saya dapat respon dari `"webhook_type": "send_message_response",`
dari pesan orang lain (sistem infra ****). 

```json
"message_type": "text",
"message": "[...CUT for PRIVACY...]",
"device_id": "******"
```

Masih di respon webhook, saya selalu dapat respon fail padahal pesan terkirim.
`"message": "Error: Send message timeout after 10000 ms.",
"status": "fail"`

**Comment by rioastamal** [comment]
Terima kasih laporan issue-nya mas @fahram . Akan segera saya investigasi. Apa `id`  dari response webhook tersebut? untuk mempercepat proses debugging.

**Reply by fahram** [reply]
"id": "2d012df0-f4ec-4e28-9a88-73337debae36",
"id": "266ef72e-16dd-4d5f-b32b-2644f139c08e",
ini ada 2 yang masuk ke webhook url saya.

**Reply by rioastamal** [reply]
mas @fahram, apakah hanya dua webhook ini yang keliru atau ada webhook response lain?

**Reply by fahram** [reply]
ada banyak bang, sepertinya memang masuk terus sesuai jadwal crontabnya (per 3 jam)

**Reply by rioastamal** [reply]
Halo mas @fahram, versi terbaru v1.0-RC11 telah dirilis. Mohon dibantu cek apakah bug tersebut masih ada setelah rilis ini. Thanks.

**Reply by fahram** [reply]
Siap.

**Reply by fahram** [reply]
Terakhir webhooknya yang nyasar `2021-11-21 10:05:08 `
sepertinya sudah **OK**

**Reply by rioastamal** [reply]
Sip. Thank you mas @fahram. Jika menemukan masih ada bug dikemudian hari tolong infokan.

**Reply by fahram** [reply]
ada satu masuk bang 
```
[2021-11-21 18:00:08]
id      : kwid-*****
```

**Reply by rioastamal** [reply]
Ok terima kasih, akan saya coba lihat.
